hub-js/graphql/local/schema.graphql

```diff
@@ -123,7 +123,7 @@ type TypeInstanceResourceVersionSpec {
                 abstract: backendRef.abstract,
                 fetchInput: {
                    typeInstance: { resourceVersion: rev.resourceVersion, id: ti.id },
-                   backend: { context: backendCtx.context, id: backendRef.id}
+                   backend: { context: apoc.convert.fromJsonMap(backendCtx.context), id: backendRef.id}
```

Comment:
Question: why is this change necessary? (I'm just curious 🙂 )

Reply:
Tests were failing without that, after merging main into this branch these files will not be visible.